test/data/testSP.py

```diff
@@ -421,36 +416,34 @@ def testmetadata_input_file(self):
         # must end in `dal`.
         assert "dvb_sp.in" in self.data.metadata["input_file_name"]
 
-    @skipForParser('GAMESSDAT', 'The parser is still being developed so we skip this test')
     def testmetadata_methods(self):
         """Does metadata have expected keys and values?"""
         # TODO implement and unify across parsers; current values are [],
         # ["HF"], ["RHF"], and ["DFT"]
         assert "methods" in self.data.metadata
 
-    @skipForParser('GAMESSDAT', 'The parser is still being developed so we skip this test')
     def testmetadata_package(self):
         """Does metadata have expected keys and values?"""
         # TODO How can the value be tested when the package name comes from
         # the parser and isn't stored on ccData?
         assert "package" in self.data.metadata
 
     @skipForParser('FChk', 'Formatted Checkpoint files do not have section for legacy package version')
-    @skipForParser('GAMESSDAT', 'Parser still in development')
+    @skipForParser('GAMESSDAT', 'Files do not contain information about the legacy package version')
     def testmetadata_legacy_package_version(self):
         """Does metadata have expected keys and values?"""
         # TODO Test specific values for each unit test.
         assert "legacy_package_version" in self.data.metadata
 
     @skipForParser('FChk', 'Formatted Checkpoint files do not have section for package version')
-    @skipForParser('GAMESSDAT', 'The parser is still being developed so we skip this test')
+    @skipForParser('GAMESSDAT', 'Files do not contain information about the package version')
     def testmetadata_package_version(self):
         """Does metadata have expected keys and values?"""
         # TODO Test specific values for each unit test.
         assert isinstance(packaging.version.parse(self.data.metadata["package_version"]), packaging.version.Version)
 
     @skipForParser('FChk', 'point group symmetry cannot be printed')
-    @skipForParser('GAMESSDAT', 'The parser is still being developed so we skip this test')
+    @skipForParser('GAMESSDAT', 'reading point group symmetry and name is not implemented')

```

Comment:
Maybe it is because it is still truly a WIP, but as soon as you can see if something is not present so can't be parsed at all, mark it like you did for package versions, otherwise someone will come back to it later and think that it might be possible to implement, then get confused when looking at the output files.